tarantool/dbapi.py

```diff
@@ -0,0 +1,262 @@
+# -*- coding: utf-8 -*-
+from tarantool.connection import Connection as BaseConnection
+
+PARAM_STYLE = 'qmark'
+API_LEVEL = "2.0"
+THREAD_SAFETY = 1
```

Comment:
The standard requires them to be in the lower case.

Reply:
Changed to lower case and renamed properly